test/services/sequelize-column-type-getter.test.js

```diff
@@ -1,8 +1,33 @@
 const ColumnTypeGetter = require('../../services/analyzer/sequelize-column-type-getter');
 const SequelizeHelper = require('../../test-utils/sequelize-helper');
-const { DATABASE_URL_MYSQL_MAX, DATABASE_URL_POSTGRESQL_MAX } = require('../../test-utils/database-urls');
+const {
+  DATABASE_URL_MYSQL_MAX,
+  DATABASE_URL_POSTGRESQL_MAX,
+} = require('../../test-utils/database-urls');
 
 describe('services > column type getter', () => {
+  describe('handling `JSON` type', () => {
+    it('should work for MySQL and pgSQL', async () => {
```

Comment:
We do not use the `pgSQL` shortcut elsewhere so I guess we could use the full name.

```suggestion
    it('should work for MySQL and PostgreSQL', async () => {
```